### PR TITLE
Fix running unit tests on cl

### PR DESCRIPTION
**Pull request by artem-belov**
Signed-off-by: Artem Belov <artem.belov@xored.com>

## Description
Fix running unit tests from all modules

## Motivation and Context
Currently cl shows:
```
DONE 0 tests in 0.093s
```

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [x] Tested locally
- [ ] Have not tested

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: scripts/run_unit_tests.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+CUR_DIR=$(dirname "$0")
+TEST_LIST=$("$CUR_DIR"/for-each-module.sh "go list ./..." | grep -v -e "sample" | grep -e "^github")
+# shellcheck disable=SC2086
+gotestsum --junitfile ~/junit/unit-tests.xml -- -short $TEST_LIST
```
Comment (haiodo) [inline scripts/run_unit_tests.sh]: I think we need a file per folder, or it unify all results?

Comment (denis-tingajkin) [reply scripts/run_unit_tests.sh]: It is unify all results

Comment (artem-belov) [reply scripts/run_unit_tests.sh]: @denis-tingajkin is right, it is unify all results

Comment (haiodo) [reply scripts/run_unit_tests.sh]: ok

**Pull request merged**